File: app/src/main/java/yangchengyu/shmtu/edu/cn/calligraphyrecognize/listener/ItemTouchHelperListener.java
package yangchengyu.shmtu.edu.cn.calligraphyrecognize.listener;

import yangchengyu.shmtu.edu.cn.calligraphyrecognize.bean.WordInfo;

/**
 * Created by kimi9 on 2018/3/20.
 */

public interface ItemTouchHelperListener {
    //数据交换
    void onItemMove(int fromPosition, int toPosition);

    //数据删除
    void onItemDissmiss(int position);

    //恢复item
    public void onItemRecover(int position, WordInfo wordInfo);

}

File: app/src/main/java/yangchengyu/shmtu/edu/cn/calligraphyrecognize/listener/ItemTouchHelperListener.kt
package yangchengyu.shmtu.edu.cn.calligraphyrecognize.listener

import yangchengyu.shmtu.edu.cn.calligraphyrecognize.bean.WordInfo

/**
 * Created by kimi9 on 2018/3/20.
 */

interface ItemTouchHelperListener {
    //数据交换
    fun onItemMove(fromPosition: Int, toPosition: Int)

    //数据删除
    fun onItemDelete(position: Int)

    //恢复item
    fun onItemRecover(position: Int, wordInfo: WordInfo)
}
